src/platform/user/authentication/utilities.js

```diff
@@ -15,14 +17,16 @@ export const ssoKeepAliveEndpoint = () => {
 };
 
 function sessionTypeUrl(type = '', version = 'v0', application = null) {
-  const SESSIONS_URI =
+  const base =
     version === 'v1'
       ? `${environment.API_URL}/v1/sessions`
       : `${environment.API_URL}/sessions`;
+  const params = new URLSearchParams();
 
-  return `${SESSIONS_URI}/${type}/new${
-    application ? `?application=${application}` : ''
-  }`;
+  if (application) params.append('application', application);
+  if (getForceAuth() && version === 'v1') params.append('force', 'true');
```

Comment:
@ncksllvn / @hinzed1127 I'm having some issues testing this if block.  When sessionTypeUrl is executed in my browser (after I click the "Sign in with ID.me" button), the `getForceAuth` var is undefined.  Any ideas to why the import statement on line 8 isn't working?

Reply:
I figured out the problem with `getForceAuth` my arrow function syntax was incorrect and because it was wrapped in curly braces I needed an explicit return.